src/Extractor/Extractor.php

```diff
@@ -154,6 +155,7 @@ public function runProperties(array $query, array $properties): array
             $this->logger->info(sprintf("Running query '%s'", $query['outputTable']));
 
             foreach ($properties as $property) {
+                $this->logger->info(sprintf('Property "%s" started.', $property['propertyName']));
```

Comment:
trochu mi ta veta nedava zmysel, co tak `Property "" export started`?

Reply:
to iste aj pre profile

Reply:
jo, jasně že export :)